ci/check-version-file.sh

```diff
@@ -24,12 +24,12 @@ do
     # If this commit is part of a PR merging into one of the protected branches, make sure we are on a pinned engine version.
     # IMPORTANT: For this to work, make sure that a new build is triggered in buildkite when a PR is opened. (This is a pipeline setting in the buildkite web UI)
     # If not, buildkite may re-use a build of this branch before the PR was created, in which case the merge target was not known, and this check will have passed.
-    if [ "$BUILDKITE_PULL_REQUEST_BASE_BRANCH" == "$item" ]; then
+    if [ "${BUILDKITE_PULL_REQUEST_BASE_BRANCH}" == "${item}" ]; then
```

Comment:
nit: think we prefer to use `[[ .. ]]` instead of `[ ... ]` as it makes the parsing of some variables easier 

(and capitalize `item`)